openlibrary/macros/SocialShare.html

```diff
@@ -5,27 +5,76 @@
-  <ul class="shareLinks-list">
-    $for share_link in share_links:
-      $ track_tag = share_link['text'].replace(' ', '-')
-      <li>
-        <a href="$share_link['url']" class="sansserif large" target="_blank" data-ol-link-track="Share|$track_tag">
-          <img title="Share on $(share_link['text'])" alt="$(share_link['text'])" class="share-link" src="$static_url('images/%s.svg' % share_link['text'].lower())"/>
-        </a>
-        <div class="share-source">$share_link['text']</div>
-      </li>
-
-    <li>
-      $# rm special characters from url such as single quote which break js prompt/iframe
-      $ clean_url = page_url.replace("'", "\\'")
-      $ iframe_html = str(embed_iframe(clean_url)).strip()
-      <a class="embed-work-btn" title="$_('Embed this book in your website')" data-ol-link-track="Share|Embed"
-         onclick="prompt('Copy embed code to clipboard:', '$iframe_html');">
-        <img alt="Embed icon" class="share-link" src="$static_url('images/embed.png')">
+
+
+  <div>
+    <p style="display: inline-block" class="cta-section-title">$_('Share this book')
+
+    </p>
+
+    <ul class="shareLinks-list" style="display: inline-block; margin-left: 50px; margin-bottom: 4px;">
+      <a class="view-modal">
+          <img class="share-btn" src="../../static/images/icons/share.svg"/>
       </a>
-      <div class="share-source">$_('Embed')</div>
-    </li>
-  </ul>
+    </ul>
+
+  </div>
+
 </div>
+
+
+    <div class="popup">
+
+      <div class="content">
+        <header class="header-share">
+          <p>Share via</p>
+            <div class="close">
+              <i class="uil uil-times"></i>
+            </div>
+        </header>
+        <ul class="icons">
+
+          $for share_link in share_links:
+          $ track_tag = share_link['text'].replace(' ', '-')
+
+              <a href="$share_link['url']" class="sansserif large" target="_blank" data-ol-link-track="Share|$track_tag">
+                <img title="Share on $(share_link['text'])" alt="$(share_link['text'])" class="share-link" src="$static_url('images/%s.svg' % share_link['text'].lower())"/>
+              </a>
+
+
+
+        $# rm special characters from url such as single quote which break js prompt/iframe
+        $ clean_url = page_url.replace("'", "\\'")
+        $ iframe_html = str(embed_iframe(clean_url)).strip()
+        <a class="embed-work-btn" title="$_('Embed this book in your website')" data-ol-link-track="Share|Embed"
+          onclick="prompt('Copy embed code to clipboard:', '$iframe_html');">
+          <img alt="Embed icon" class="share-link" src="$static_url('images/embed.png')">
+        </a>
+
+
+        </ul>
+        <p>Or copy link</p>
+        <div class="copy-field">
+          <i class="url-icon uil uil-link"></i>
+          <input type="text" readonly value="$page_url">
+          <button class="copy-button">Copy</button>
+        </div>
+      </div>
+    </div>
+
+
+<script>
+
+
+
+</script>
```

Comment:
```suggestion
```
We have an ongoing effort to remove remaining inline scripts, so we would like to avoid new inline scripts when possible.